AnkiDroid/src/main/java/com/ichi2/anki/NoteEditorFragment.kt

```
@@ -1878,6 +1880,25 @@ class NoteEditorFragment :
         noteEditorViewModel.updateFieldValue(i, TextFieldValue(text = newText ?: ""))
     }
 
+
+    private fun saveNoteWithSchemaConfirmation() {
+        launchCatchingTask {
+            try {
+                saveNote()
+            } catch (e: ConfirmModSchemaException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)